lighthouse-core/lib/emulation.js

```diff
@@ -63,32 +63,47 @@ const NO_CPU_THROTTLE_METRICS = {
 };
 
 /**
+ *
  * @param {Driver} driver
- * @return {Promise<void>}
```

Comment:
could leave a `@return {Promise<void>}` on the new function FWIW :)

Reply:
done